lib/BloqadeQMC/src/ising/Rydberg.jl

```diff
@@ -1,23 +1,23 @@
 using Base.Iterators
 
-
 abstract type AbstractRydberg{O <: AbstractOperatorSampler} <: AbstractLTFIM{O} end
 
-struct Rydberg{O,M <: UpperTriangular{Float64},UΩ <: AbstractVector{Float64}, Uδ <: AbstractVector{Float64}, L <: Lattice} <: AbstractRydberg{O}
+struct Rydberg{O,M <: AbstractMatrix{Float64},UΩ <: AbstractVector{Float64}, Uδ <: AbstractVector{Float64}, A} <: AbstractRydberg{O}
     op_sampler::O
     V::M
```

Comment:
put a comment here explain what this field does?

Reply:
Added a comment explaining it is the interaction matrix :+1: